packages/shopify-codemod/test/fixtures/convert-default-export-objects-to-named-exports/literals.input.js

```diff
@@ -0,0 +1,9 @@
+export default {
```

Comment:
There should also be a test (and a bail condition) for properties that are functions that refer to `this`.